templates/docs/examples/patterns/tabs/content.html

```diff
@@ -1,54 +1,33 @@
 {% extends "_layouts/examples.html" %}
+{% from "_macros/tabs.jinja" import tabs %}
+
 {% block title %}Tabs / Tabbed Content{% endblock %}
 
 {% block standalone_css %}patterns_tabs{% endblock %}
 
 {% block content %}
-<div class="p-tabs">
-  <div class="p-tabs__list" role="tablist" aria-label="Juju technology">
-    <div class="p-tabs__item">
-      <button
-        class="p-tabs__link"
-        role="tab"
-        aria-selected="true"
-        aria-controls="olm-tab"
-        id="olm"
-      >Charmed Operator Lifecycle Manager</button>
-    </div>
-    <div class="p-tabs__item">
-      <button
-        class="p-tabs__link"
-        role="tab"
-        aria-selected="false"
-        aria-controls="operator-tab"
-        id="operator"
-        tabindex="-1"
-      >Charmed Operator SDK</button>
-    </div>
-    <div class="p-tabs__item">
-      <button
-        class="p-tabs__link"
-        role="tab"
-        aria-selected="false"
-        aria-controls="charmhub-tab"
-        id="charmhub"
-        tabindex="-1"
-      >Charmhub</button>
-    </div>
-  </div>
-
-  <div tabindex="0" role="tabpanel" id="olm-tab" aria-labelledby="olm">
-    <p>A system to help you move from configuration management to application management across your hybrid cloud estate - through sharable, reusable, tiny applications called Charmed Operators.</p>
-  </div>
+{% call(content) tabs(buttons=['Charmed Operator Lifecycle Manager', 'Charmed Operator SDK', 'Charmhub'], aria_label='Juju technology') %}
```

Comment:
OK, nice. This is promising.
I guess this is the closest we can get to "slots" in macro.

But the notation worries me a bit. The purpose of this work is to make it easier for (potentially junior) web devs, so that they don't have to maintain complex HTML structures.
Now we are asking them to maintain complex macro calls using a notation that is likely even more alien to them than HTML.

Reply:
I guess one suggestion I would have is to rename `content` to `slot` (the naming used in web components). So this may turn to be a bit more "familiar".

```
{% call(slot) some_component_macro() %}
  {% if slot = 'header' %}…{% endif %}
  {% if slot = 'footer' %}…{% endif %}
{% endcall %}
```

or, if we can pass flags instead of strings from macro with `caller({ 'header': True })`.

```
{% call(slot) some_component_macro() %}
  {% if slot.header %}…{% endif %}
  {% if slot.footer %}…{% endif %}
{% endcall %}
```